### PR TITLE
0.27.3 Software release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software_versioned_docs/version-0.27/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,24 @@ Astronomer v0.27 is the latest Stable version of Astronomer Software, while v0.2
 
 We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## 0.27.3
+
+Release date: Feb 22, 2022
+
+### Limit Workspace Creation to System Admins
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.27/release-notes.md:22]: @jwitz this is not the default behavior but only an option provided for our software customers.

File: software_versioned_docs/version-0.27/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,24 @@ Astronomer v0.27 is the latest Stable version of Astronomer Software, while v0.2
...
+
+To provide more control over resource usage across your organization, System Admins are now the only role that can create new Workspaces.
+
+For more information about the permissions that different roles have, including instructions for how to modify a role's permissions, see [Platform User Management](manage-platform-users.md).
+
+### Additional improvements
+
+- System Admins are now the only role that can create a Deployment with a deprecated version of Astronomer Certified.
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.27/release-notes.md]: I think this is also feature flag driven. @aliotta can you please share with @jwitz and also review these release notes.

File: software_versioned_docs/version-0.27/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,24 @@ Astronomer v0.27 is the latest Stable version of Astronomer Software, while v0.2
...
+
+### Limit Workspace Creation to System Admins
+
+To provide more control over resource usage across your organization, Workspace creation has been limited to users with System Admin permissions. 
+
+For more information about each role's permissions, including instructions for how to modify a role's permissions, see [Platform User Management](manage-platform-users.md).
+
+### Additional improvements
+
+- System Admins are now the only users that can create a Deployment with a deprecated version of Astronomer Certified.
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.27/release-notes.md:30]: I think this is also feature flag driven. @aliotta can you please share with @jwitz and also review these release notes.

File: software_versioned_docs/version-0.27/release-notes.md (hunk 1 above)
Comment (jwitz) [reply software_versioned_docs/version-0.27/release-notes.md:22]: Is this configurable in the UI now? I thought that this was always an option via Helm

File: software_versioned_docs/version-0.27/release-notes.md (hunk 3 above)
Comment (jwitz) [reply software_versioned_docs/version-0.27/release-notes.md:30]: I don't see an 0.27.3 tag on GH, so any links to for this specific issue would be great

File: software_versioned_docs/version-0.27/release-notes.md (hunk 1 above)
Comment (vishwas-astro) [reply software_versioned_docs/version-0.27/release-notes.md:22]: It is an option in Helm only still.

File: software_versioned_docs/version-0.27/release-notes.md (hunk 3 above)
Comment (aliotta) [reply software_versioned_docs/version-0.27/release-notes.md]: @jwitz the feature flag is houston.config.deployments.enableSystemAdminCanCreateDeprecatedAirflows

It must be set to true to enable it.

Here's the PR for that one https://github.com/astronomer/houston-api/pull/936

**Review by vishwas-astro — APPROVED** [review]

**Pull request merged**